src/lgr_advanced/lgr_renderer/urls.py

```diff
@@ -9,6 +9,7 @@
 register_converter(LgrSlugConverter, 'lgr')
 
 urlpatterns = [
+    path('<str:format>/<int:lgr_pk>', views.LGRRendererDbView.as_view(), name='lgr_render_from_db'),
```

Comment:
if this is used to render admin rz lgr, lgr_idn_admin_display_rz_lgr is already doing exactly that for html, adding a query parameter with format and xml support would do the trick